pkg/local/local.go

```diff
@@ -54,19 +54,19 @@ func InitDataVolumeDir(dataVolume string) error {
 	return os.MkdirAll(dataPath, 0777)
 }
 
-func GetLatestImages(useGpu bool) ([]string, error) {
+func GetLatestImages(useGpu bool) ([]string, string, error) {
```

Comment:
you can use the named return to be clear, It is important when you return multiple values
```
func GetLatestImages(useGpu bool) (images []string, engineImage string, err error) {
```